src/sampling.jl

```diff
@@ -463,7 +463,7 @@ seqsample_d!(a::AbstractArray, x::AbstractArray) = seqsample_d!(default_rng(), a
 """
     sample([rng], a, [wv::AbstractWeights])
 
-Select a single random element of `a`. Sampling probabilities are proportional to
+Select a single random element of array `a`. Sampling probabilities are proportional to
```

Comment:
Alternatively, we should adjust the signature above to match the actual signature

Reply:
you decide!

Reply:
Then let's go with that. Would you mind doing it here?